src/licensedcode/data/rules/not-a-license_434.RULE

```diff
@@ -0,0 +1 @@
+S5PC100_GPL0(0)
```

Comment:
Was this really detected?

Reply:
No. On second thought this is really unnecessary, removing and renaming these.